code42/py42
# (bugfix) nil properties

### Description of Change ###

Some device settings have the format {"@nil": "true"} if they have not previously been set. This updates the getter and setter in the base property to properly handle this format.

Additionally, I added the javaMemoryHeapMax property, which is an example that commonly uses this format.

### Issues Resolved ###
n/a, I did not file an issue around this.

### Testing Procedure ###
1. Create an SDK object
2. Retrieve device settings for a device in Code42 that has not previously altered the javaMemoryHeapMax property
3. Ensure that this property can be properly viewed

### PR Checklist ###
Did you remember to do the below?

- [n/a] Add an entry to CHANGELOG.md describing this change (it does not look like we document a list of available properties anywhere, which is the only user-facing change made here)
